release/resources/module_list.json

```diff
@@ -1,377 +1,288 @@
         },
         {
-            "name": "module-ballerina-url",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-url"
         },
         {
-            "name": "module-ballerina-uuid",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-uuid"
         },
         {
-            "name": "module-ballerina-websocket",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-websocket"
         },
         {
-            "name": "module-ballerina-websub",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-websub"
         },
         {
-            "name": "module-ballerina-websubhub",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-websubhub"
         },
         {
-            "name": "module-ballerina-xmldata",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-xmldata"
         },
         {
-            "name": "module-ballerina-xslt",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-xslt"
         },
         {
-            "name": "module-ballerina-yaml",
-            "display_code_cov_badge": true
+            "name": "module-ballerina-yaml"
         }
     ],
     "extended_modules": [
         {
-            "name": "module-ballerinax-copybook",
-            "release": false,
-            "display_code_cov_badge": true
+            "name": "module-ballerinax-copybook"
         },
         {
             "name": "module-ballerinax-persist.googlesheets",
-            "version_key": "stdlibPersistGoogleSheetVersion",
-            "display_code_cov_badge": true
+            "version_key": "stdlibPersistGoogleSheetVersion"
         },
         {
             "name": "module-ballerinax-persist.inmemory",
-            "version_key": "stdlibPersistInmemoryVersion",
-            "display_code_cov_badge": true
+            "version_key": "stdlibPersistInmemoryVersion"
         },
         {
             "name": "module-ballerinax-persist.redis",
-            "version_key": "stdlibPersistRedisVersion",
-            "display_code_cov_badge": true
+            "version_key": "stdlibPersistRedisVersion"
         },
         {
             "name": "module-ballerinax-persist.sql",
-            "version_key": "stdlibPersistSqlVersion",
-            "display_code_cov_badge": true
+            "version_key": "stdlibPersistSqlVersion"
         },
         {
             "name": "module-ballerinax-wso2.controlplane",
-            "version_key": "stdlibWso2ControlplaneVersion",
-            "display_code_cov_badge": true
+            "version_key": "stdlibWso2ControlplaneVersion"
         }
     ],
-    "connectors": [
+    "handwritten_connectors": [
         {
-            "name": "module-ballerinax-asana",
-            "display_code_cov_badge": false
+            "name": "module-ballerinax-asana"
```

Comment:
This should come under `generated_connectors`

Reply:
The plan was to ask team members to update the JSON once we finalize this. The dashboard will be generated accordingly. Shall we move ahead in that direction?

Reply:
@ThisaruGuruge Ack. +1